unified-installer/astra-unified-installer.sh

```diff
@@ -2133,136 +2340,14 @@ step_init_generated_dirs_and_files() {
 
     rm -rf "$kustomization_dir" "$crs_dir"
     mkdir -p "$crs_dir"
-    mkdir -p "$kustomization_dir"
 
-    if [ ! -f "$kustomization_file" ]; then
-        cat <<EOF > "$kustomization_file"
-apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
-resources:
-patches:
-transformers:
-EOF
-        logdebug "$kustomization_file: OK"
-    fi
-}
```

Comment:
The block above has been moved to its own function, `generate_kustomization_dirs_and_files`.